internal/utils/comma_int.go

```
@@ -0,0 +1,17 @@
+
+import (
+	"fmt"
+	"io"
+)
+
+type CommaInt int64
+
+func (c CommaInt) Format(f fmt.State, verb rune) {
+	switch verb {
+	case 'd', 'v':
+		io.WriteString(f, FormatIntWithCommas(int64(c)))
```

golangci-lint (1.64.8)

Error return value of `io.WriteString` is not checked

(errcheck)